src/openforms/scss/components/admin/_index.scss

```
@@ -38,3 +39,6 @@
 
 // Submission logs
 @import './logic-evaluated-table';
+
+// Errors
+@import 'error-message';
```

```suggestion
@import './error-message';
```
bit of consistency :)